jest.config.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+const dotenv = require('dotenv');
+dotenv.config();
+
+export default {
+  transform: {
+    '^.+\\.ts?$': 'ts-jest'
+  },
+  testTimeout: 10000,
+  clearMocks: true,
+  collectCoverage: true,
```

Comment:
```suggestion
  collectCoverage: true,
  collectCoverageFrom: [
    './**/*.ts',
    '!./**/tests/**/*.ts',
    '!./tests/**/*.ts',
    '!./jest.config.ts'
  ],
```
can we add a `collectCoveregeFrom` directive? this makes the coverage metric more accurate....

Reply:
added